### PR TITLE
Fix NPE

### PR DESCRIPTION
**Pull request by Ifropc**

### CODE REVIEW

**Review by leighmcculloch — APPROVED** [review]
👏🏻 Nice catch.

File: src/test/java/org/stellar/sdk/KeyPairTest.java [hunk 1]
```diff
@@ -120,4 +120,11 @@ public void testSignPayloadSignerLessThanHint() {
     // the hint could only be derived off of 3 bytes from payload
     Assert.assertArrayEquals(sig.getHint().getSignatureHint(), new byte[]{(byte)(255), 64, 7, 55});
   }
+
+  @Test
+  public void testPublicEqual() {
+    KeyPair keypair = KeyPair.fromAccountId("GDEAOZWTVHQZGGJY6KG4NAGJQ6DXATXAJO3AMW7C4IXLKMPWWB4FDNFZ");
+    KeyPair keypairCopy = KeyPair.fromAccountId("GDEAOZWTVHQZGGJY6KG4NAGJQ6DXATXAJO3AMW7C4IXLKMPWWB4FDNFZ");
+    Assert.assertEquals(keypairCopy, keypair);
+  }
```
Comment (leighmcculloch) [inline src/test/java/org/stellar/sdk/KeyPairTest.java:129]: Thanks for adding a test for the case being fixed. I don't think there's a test for equals in the cases that one keypair is private and one is public, or that both are private. If those tests don't exist elsewhere that I'm missing, could we add them too?

**Review by leighmcculloch — APPROVED** [review]
👏🏻 Nice catch.

File: src/test/java/org/stellar/sdk/KeyPairTest.java (hunk 1 above)
Comment (Ifropc) [reply src/test/java/org/stellar/sdk/KeyPairTest.java:129]: Done

**Review by leighmcculloch — APPROVED** [review]
🎉

**Pull request merged**